pkg/grpc/api.go

```diff
@@ -1263,7 +1263,13 @@ func (a *api) GetConfigurationAlpha1(ctx context.Context, in *runtimev1pb.GetCon
 		Metadata: in.Metadata,
 	}
 
-	getResponse, err := store.Get(ctx, &req)
+	policy := a.resiliency.ComponentPolicy(ctx, in.StoreName)
+	getResponse := &configuration.GetResponse{}
+	err = policy(func(ctx context.Context) error {
```

Comment:
Slightly more succinct.

```golang
	err = policy(func(ctx context.Context) (err error) {
		getResponse, err := store.Get(ctx, &req)
		return getResponse, err
	})
```

Reply:
Changed to what the previous comment states. But unless I'm misunderstanding, can't we only return 1 thing from this because that's how the `resiliency.Operation` is defined?